crds/vshn.appcat.vshn.io_xvshnminios.yaml

```diff
@@ -340,50 +340,120 @@ spec:
             type: object
           status:
             properties:
-              spec:
-                description: ResourceStatus represents the observed state of a managed
-                  resource.
+              conditions:
```

Comment:
Is there a reason to change Minio CRD in this PR?

Reply:
This is an unrelated change. Unfortunately it was forgotton to run `make generate` in a previous PR